HelpSource/Classes/T2K.schelp

```diff
@@ -10,7 +10,13 @@ classmethods::
 method:: kr
 
 argument:: in
-input signal.
+An audio rate input signal.
+
+discussion::
+note::
+During initialization of the UGen graph, teletype::T2K:: will not forward a trigger to downstream UGens if a trigger is received after the first sample in the first block.
+Once running, however, its first output sample will be a trigger if one is received within the first block.
```

Comment:
I'm having a hard time understanding these two sentences. Let's perhaps try to rephrase it and add a couple of examples?

Reply:
Thanks! I've updated the description, hopefully this reads more clearly.

Regarding examples, I don't think it's necessary, as it's quite a fringe case, and in the rare case that anyone encounters it, this Discussion would be sufficient to tell them what's going on IMO. Let me know though if you disagree!